src/textual/css/stylesheet.py

```diff
@@ -86,18 +86,18 @@ def read(self, filename: str) -> None:
                 css = css_file.read()
             path = os.path.abspath(filename)
         except Exception as error:
-            raise StylesheetError(f"unable to read {filename!r}; {error}") from None
+            raise StylesheetError(f"unable to read {filename!r};") from error
```

Comment:
This particular error should never render as a traceback, so I'd prefer to keep it with the semi-colon delimited errors.